package/files/nginx/nginx-http.conf

```diff
@@ -17,12 +18,28 @@ server {
     }
 
     location = /repo/repoindex.xml {
-        return 301 https://$host$request_uri;
```

Comment:
In the cloud that would have upgraded the existing instances nicely from HTTP to HTTPS, and now they'd be stuck forever on HTTP. I guess now we need a separate config for the cloud :thinking: